src/sentry/utils/redis.py

```diff
@@ -287,6 +278,14 @@ def call_script(client, keys, args):
             script[0].registered_client = None
         return script[0](keys, args, client)
 
+    call_script.__doc__ = u"""Executes {!r} as a Lua script on a Redis server.
+
+Takes the client to execute the script on as the first argument,
+followed by the values that will be provided as ``KEYS`` and ``ARGV``
+to the script as two sequence arguments.""".format(
+        path
+    )
+
```

Comment:
What's the problem?

Reply:
Seems much less readable to me, why would we move the comment out of the function?

Reply:
Well, it's wrong inside the function like that https://github.com/getsentry/sentry/issues/21139

Reply:
I'd rather remove the string formatting then. I'm not sure there's anywhere that we actually display this docstring in our codebase? cc @tkaemming

Reply:
Cool, I agree with that

Reply:
> I'm not sure there's anywhere that we actually display this docstring in our codebase? 

Not to my knowledge. This would only be useful in autogenerated docs (i.e. `help(load_script(…))`.)

Reply:
I'll let @tkaemming have the final opinion here.

Reply:
Okay, I'll just convert it into a comment then.

Reply:
👍 Matt also had a suggestion, but IMO we should just remove the dynamicness of it